handlers/docker/docker_client.go

```diff
@@ -1,4 +1,4 @@
-package dockerClient
+package docker
```

Comment:
I can't comment on lower lines that you didnt change in this PR so I'm commenting on this line, but:

Don't set an environemnt variable for version like you did here:
```
envErr := os.Setenv("DOCKER_API_VERSION", version)
```
The docker client has a method `UpdateClientVersion()`, so just get the client and the update the version.

Also, I noticed that the python agent vesion has the following signature:
```
def docker_client(version=None, base_url_override=None, tls_config=None,
                  timeout=None):
```
Of those parameters, only version and timeout are used in the python code. So, you can drop base_url_override and tls_config, but need to implement timeout.